boards/esp32-c3-12f.json

```diff
@@ -0,0 +1,36 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32c3_out.ld"
+    },
+    "core": "esp32",
+    "f_cpu": "160000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "dio",
+    "extra_flags": [
+      "-DARDUINO_USB_MODE=1"
+    ],
+    "mcu": "esp32c3",
+    "variant": "esp32c3"
+  },
+  "connectivity": [
+    "wifi"
+  ],
+  "debug": {
+    "openocd_target": "esp32c3.cfg"
+  },
+  "frameworks": [
+    "arduino",
+    "espidf"
+  ],
+  "name": "Ai-Thinker ESP32-C3-12F",
+  "upload": {
+    "flash_size": "2MB",
```

Comment:
Am i correct that the "flash_size" and "name" lines are the only difference to `esp32-c3-devkitm-1`

maybe to make the whole thing more generic, lets rename it to something like esp32-c3-devkitm_2M?
@blazoncek what do you think?

Reply:
Flash size may be different on many boards and add to that that some may feature PSRAM as well.
It will be inevitable to add as many build environments as there are possible boards *or* provide instructions how users can create their own environment in PIO override.

Reply:
I think the 2M partition table may be usefull for others too, because currently we don't have a WLED partions.csv for mcu's with 2M flash only. @andyshinn is OTA still possible with 2M?

Reply:
> @andyshinn is OTA still possible with 2M?

Not for C3, S2, S3 or regular ESP32